src/modules/storage/LedgerStorage.ts

```diff
@@ -374,8 +375,11 @@ export class LedgerStorage extends Storages
                     pre_image.distance,
                     pre_image.distance
                 ])
-                .then(() =>
+                .then((result) =>
                 {
+                    if (result.changes == 1)
+                        logger.info(`Saved a pre-image enroll_key : ${pre_image.enroll_key.toString().substr(0, 18)}, ` +
+                        `hash : ${pre_image.hash.toString().substr(0, 18)}, distance : ${pre_image.distance}`);
```

Comment:
I would like to be able to set the permissions for log output in Stoa

Reply:
Yes, that would be good.
I applied it now.